### PR TITLE
52 deploy page

### LINKED ISSUE #52: Deploy page
Add a page for deployment.

For the first iteration, a text input with a deploy btn would be enough.

### PR DESCRIPTION
**Pull request by PaulFarault**
#### Which issue(s) this PR fixes

Fixes #52 

#### Agreements

- [X] I hereby declare this contribution to be licensed under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0).
- [X] I hereby agree to grant [TOSIT](https://www.tosit.io/) a copyright license to use my contributions.

### CODE REVIEW

**Review by gboutry — CHANGES_REQUESTED** [review]
Input targets should not be re-ordered on modification

`,,` multiple colons input are possible while they shouldn't

**Comment by gboutry** [comment]
The input text is nearly invisible. If you don't know it's there, it's hard

File: src/app/deploy/InputField.tsx [hunk 1]
```diff
@@ -11,14 +12,31 @@ export default function InputField() {
...
-      <input
-        type="text"
-        value={inputValue}
-        onChange={(e) => setInputValue(e.target.value)}
-        className="flex-grow"
-      />
-      <button type="submit">Add</button>
+    <form onSubmit={handleOnSubmit}>
+      <label
+        htmlFor="email"
```
Comment (gboutry) [inline src/app/deploy/InputField.tsx:17]: De we need an email address to deploy tdp ?

Comment (PaulFarault) [reply src/app/deploy/InputField.tsx:17]: Thx! It should be better now.

Comment (gboutry) [reply src/app/deploy/InputField.tsx:17]: the properti htmlFor still has the value `htmlFor`, which is wrong

**Review by jpsavy — APPROVED** [review]
A zookeeper_config has been successfully deployed.

**Review by gboutry — APPROVED** [review]

**Pull request merged**